contracts/erc7683/ERC7683.sol

```diff
@@ -6,10 +6,10 @@ pragma solidity ^0.8.0;
 struct GaslessCrossChainOrder {
     /// @dev The contract address that the order is meant to be settled by.
     /// Fillers send this order to this contract address on the origin chain
-    address originSettler;
+    bytes32 originSettler;
     /// @dev The address of the user who is initiating the swap,
     /// whose input tokens will be taken and escrowed
-    address user;
+    bytes32 user;
```

Comment:
Aren't these local addresses? Why would they need to be bytes32?

Reply:
Mainly, consistency with the rest of the data, as this would require extra casting at some point. Your point is that everything that is exclusively local to the evm should remain an address and be cast when needed (for example, in events?), so having a mixed model where we have both addresses and bytes32 seems inconsistent. Here, we have intentionally converted everything to bytes32 for greater consistency. I think we should discuss a standard criteria with the team for these changes.

Reply:
yeah this is a really good point @mrice32 , and it's how we designed ERC7683 to work too: where code that is expected to run on EVM doesn't need to convert addresses into bytes32. If we undid all these address=>bytes32 changes, would we remove some breaking changes? That very well might be worth it